client/src/reducers/log_in_user_reducer.js

```diff
@@ -23,7 +23,7 @@ function logInUserReducer(state = initialState, action) {
       return {
         ...state,
         data: "",
-        loggedIn: action.payload
+        loggedIn: ""
```

Comment:
you need to return "action.payload"

Reply:
For some reason, continue as guest was still looking for loggedin username, this is just a quick fix

Reply:
well can you fix it back with using action.payload?